test/acceptor_SUITE.erl

```diff
@@ -1202,8 +1202,20 @@ supervisor_changed_options_restart(_) ->
 	{ok, [{send_timeout, 300001}]}
 		= inet:getopts(do_get_listener_socket(ListenerSupPid1), [send_timeout]),
 	%% Crash the listener_sup process, allow a short time for restart to succeed.
-	exit(ListenerSupPid1, kill),
-	timer:sleep(1000),
+	ListenerChilds = [ChildPid || {_, ChildPid, _, _} <- supervisor:which_children(ListenerSupPid1)],
+	FilterFun = fun (#{meta := #{pid := EventPid}}, _) ->
+		case lists:member(EventPid, ListenerChilds) of
+			true -> stop;
+			false -> ignore
+		end
+	end,
+	ok = logger:add_primary_filter(?MODULE, {FilterFun, undefined}),
+	try
+		exit(ListenerSupPid1, kill),
+		receive after 1000 -> ok end
```

Comment:
Let's just write `timer:sleep(1000)`.

Reply:
Ok, done. I did that because `receive` is used for waiting almost everywhere else.